app/views/mail_notifications/call_for_evidence_reminder.text.erb

```diff
@@ -0,0 +1,6 @@
+This a reminder that the call for evidence "<%= @title %>" closed 8 weeks ago and you may want to publish responses. You can do this in the Outcome section.
+
+It is optional to publish responses for a call for evidence.
+
+Read how to write good call for evidences pages here:
+{placeholder for link}
```

Comment:
A few (small!) things:

1. I think the first sentence is missing the word `"is"`.
2. Could we remove the entire reference to Call for Evidence guidance, since we don't have the link yet? So we don't accidentally launch with a `{placeholder}`. It seems like the email will still be just as useful without that placeholder reference for now.
3. It might be nice to add a link to the document, either on GOV.UK or in Whitehall. Other emails ([example](https://github.com/alphagov/whitehall/blob/0e0ffc8ec6b4667bacecb17bfe9bba55e3e2581f/app/views/mail_notifications/edition_rejected.text.erb), [example](https://github.com/alphagov/whitehall/blob/0e0ffc8ec6b4667bacecb17bfe9bba55e3e2581f/app/views/mail_notifications/edition_published.text.erb)) include a link to the document – although I realise the consultation ones don't ([example](https://github.com/alphagov/whitehall/blob/0e0ffc8ec6b4667bacecb17bfe9bba55e3e2581f/app/views/mail_notifications/consultation_deadline_passed.text.erb)), so it's probably fine to leave it as is.

```suggestion
This is a reminder that the call for evidence "<%= @title %>" closed 8 weeks ago and you may want to publish responses. You can do this in the Outcome section.

It is optional to publish responses for a call for evidence.
```